pandas/core/frame.py

```diff
@@ -5723,6 +5730,10 @@ def reset_index(
             levels are named. If None then the index name is repeated.
         allow_duplicates : bool, optional, default lib.no_default
             Allow duplicate column labels to be created.
+        names : str, tuple or list, default None
+            Using the given string, rename the DataFrame column which contains the
+            index data. If the DataFrame has a MultiIndex, this has to be a list or
+            tuple with length equal to the number of levels.
```

Comment:
needs a versionadded tag (allow_duplicates has one already but you need to add a separate)

Reply:
Hi Current code already contains version `.. versionadded:: 1.5.0` at below of description. 
```
allow_duplicates : bool, optional, default lib.no_default
    Allow duplicate column labels to be created.
names : str, tuple or list, default None
    Using the given string, rename the DataFrame column which contains the
    index data. If the DataFrame has a MultiIndex, this has to be a list or
    tuple with length equal to the number of levels.

    .. versionadded:: 1.5.0
```

Reply:
Sorry. I do not understand. Are you telling me to add another new version? Based on my information, latest version is 1.5.0.

Reply:
ok. Have add `.. versionadded:: 1.5.0` to `allow_duplicates` .